test/simplifyExpression/fractionsSearch/cancelLikeTerms.test.js

```diff
@@ -2,8 +2,8 @@ const cancelLikeTerms = require('../../../lib/simplifyExpression/fractionsSearch
 
 const TestUtil = require('../../TestUtil');
 
-function testCancelLikeTerms(exprStr, expectedStr) {
-  TestUtil.testSimplification(cancelLikeTerms, exprStr, expectedStr);
+function testCancelLikeTerms(exprString, outputStr) {
```

Comment:
maybe use both - `expectedOutputStr`? cause it might not be the output, it's just what we hope will be the output